graphql/src/main/java/com/scalar/db/graphql/schema/BigIntCoercing.java

```diff
@@ -0,0 +1,72 @@
+package com.scalar.db.graphql.schema;
+
+import com.scalar.db.io.BigIntValue;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import java.math.BigInteger;
+
+/** Handles type conversions for Scalar DB {@link BigIntValue}. */
+class BigIntCoercing implements Coercing<Long, Long> {
+  static final BigIntCoercing INSTANCE = new BigIntCoercing();
+
+  private BigIntCoercing() {}
+
+  @Override
+  public Long serialize(Object dataFetcherResult) throws CoercingSerializeException {
```

Comment:
My Intellij shows a warning for it:
```suggestion
  public Long serialize(@Nonnull Object dataFetcherResult) throws CoercingSerializeException {
```

Reply:
Thank you!
I didn't add the annotation to our classes since the interface methods are annotated with `org.jetbrains.annotations.NotNull` and it depends on the JetBrains package. I didn't know that I could remove that warning by adding `javax.annotation.Nonnull`. Is it okay to mix different kinds of nullability annotations?

Reply:
Actually, we don't have any rule for it, but I usually use `javax.annotation.*` annotations in that case. @feeblefakie What do you think?

It looks like the author of this comment prefers `javax.annotation`, too:
https://stackoverflow.com/a/42695253/4746611

Reply:
Since we don't want to add a dependency to `org.jetbrains` package, `javax.annotation` seems the way to go. I added it in 4a602d5 anyway. Please leave a comment if there are any problems.

Reply:
I think, so far, we only use javax.annotation, so let's use `javax.annotation.Nonnull`.